framework/src/executioners/IterativeMultiAppSolve.C

```diff
@@ -78,26 +80,30 @@ IterativeMultiAppSolve::validParams()
                                     "relaxation_factor>0 & relaxation_factor<2",
                                     "Fraction of newly computed value to keep."
                                     "Set between 0 and 2.");
-  params.addParam<std::vector<std::string>>("transformed_variables",
-                                            std::vector<std::string>(),
-                                            "List of master app variables to transform during coupling iterations");
-  params.addParam<std::vector<std::string>>("transformed_postprocessors",
-                                            std::vector<std::string>(),
-                                            "List of master app postprocessors to transform during coupling iterations");
-  params.addDeprecatedParam<std::vector<std::string>>("relaxed_variables",
-                                                      std::vector<std::string>(),
-                                                      "Relaxed variables is deprecated, use transformed_variables instead.",
-                                                      "List of master app variables to relax during coupling iterations");
+  params.addParam<std::vector<std::string>>(
```

Comment:
AllLocalDof.. object expects a string. I tried to change everyone but then eventually hit that var.name() returns a string. So I think this will be for later.